model/minio/minio_pool.rb

```diff
@@ -12,23 +12,23 @@ class MinioPool < Sequel::Model
   include ResourceMethods
   include SemaphoreMethods
 
-  semaphore :destroy
+  semaphore :destroy, :additional_pool
```

Comment:
nit: add|create_additional_pool. Otherwise it is not clear what we are doing with the additional pool. Maybe we are destroying it.